components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__gene-variants.xml

```diff
@@ -162,18 +166,29 @@
     }
     //update genesymbol label for input field
     el.previous().innerHTML = value;
+    // update geneid label for the hidden input field
+    el.next("input.gene-id").value = id || value;
     //update info icon
     fetchGeneDetails([el],[value], isSuggestSelected);
   }
 
-  fetchGeneDetails($$('input.gene-name'), null, true);
+  // Fetch gene details, marking any non-standardized ids.
+  $$('input.gene-name').each(function(elt) {
+    if (!elt.className.endsWith('nonstandard')) {
+      fetchGeneDetails([elt], null, true);
+    } else {
+      fetchGeneDetails([elt], null, false);
+    }
```

Comment:
The last 5 lines could be replaced with `fetchGeneDetails([elt], null, !elt.className.endsWith('nonstandard'));`